pulsar-client-cpp/lib/Murmur3_32Hash.h

```diff
@@ -38,9 +38,6 @@ class Murmur3_32Hash : public Hash {
     int32_t makeHash(const std::string& key);
 
    private:
-    static constexpr int32_t CHUNK_SIZE = 4;
```

Comment:
These could still be defined as `static const int32_t` types

Reply:
@merlimat Yes! But the value definition will be in `cc` file.

Reply:
IMO, macro is better solution.